CHANGELOG.md

```diff
@@ -43,6 +44,8 @@
 
 ## Changes since v6.1.1
 
+- [#816](https://github.com/oauth2-proxy/oauth2-proxy/pull/816) (via [#936](https://github.com/oauth2-proxy/oauth2-proxy/pull/936)) Support non-list group claims (@loafoe)
```

Comment:
Never had a via before, works though!

Reply:
I wanted to give credit for discovering the issue and implementing a potential solution -- but I also didn't want to merge that PR then to immediately rip it out and refactor the concept somewhere else. This felt like good compromise.